### PR TITLE
[86encyf8z] New Table in Homepage

### PR DESCRIPTION
**Pull request by pathusutariya**
- Table Created.

### CODE REVIEW

File: index.html
Comment (cl7-ab2) [inline index.html:1]: why this page?  We need separate page for this.

File: index.html [hunk 1]
```diff
@@ -7,5 +7,24 @@
...
+	<tr>
+		<td>1</td>
+		<td>Dharmik</td>
+		<td>dharmikp@codelab7.com</td>
+		<td>9876543210</td>
+	</tr>
+	<tr>
+		<td>2</td>
+		<td>Ankur Bharodiya</td>
+		<td>ankurb@codelab7.com</td>
```
Comment (cl7-ab2) [inline index.html:26]: Phone number missing for Ankur

File: index.html [hunk 2]
```diff
@@ -7,5 +7,24 @@
...
+	</tr>
```
Comment (cl7-ab2) [inline index.html:28]:
```suggestion
<td>7896465321</td>
	</tr>
```

**Review by cl7-ab2 — CHANGES_REQUESTED** [review]
- Table missing

File: index.html
Comment (pathusutariya) [reply index.html:1]: Because we need it on homepage.

**Review by pathusutariya — COMMENTED** [review]
All changes are done.

**Review by cl7-ab2 — APPROVED** [review]
Good to go

**Pull request merged**